### PR TITLE
Added Eclipse formatter

### LINKED ISSUE #91: Code formatting settings in IDE's
Since different developers are working on the project, the code formatting in IDE's should be standardized.

IDE's:
- Eclipse
- IntelliJ

A common method is to use the Eclipse formatting scheme, which can be imported into an IntelliJ project.

https://github.com/redhat-developer/vscode-java/blob/master/formatters/eclipse-formatter.xml
https://blog.jetbrains.com/idea/2014/01/intellij-idea-13-importing-code-formatter-settings-from-eclipse/

### PR DESCRIPTION
**Pull request by mzudse**
This closes #91

### CODE REVIEW

**Comment by astanik** [comment]
@calipee Can you please make your review for this issue?

**Review by calipee — CHANGES_REQUESTED** [review]

**Comment by calipee** [comment]
> @calipee Can you please make your review for this issue?

I added the comment but forgot to submit my review, I was already wondering why nobody replied for a week 😅

File: formatter/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Formatting
+
+This directory contains the Eclipse formatting configuration files used to enforce the same code style on all contributors.
+
```
[the remark this answers is not in the document]
Comment (mzudse) [reply formatter/README.md:4]: Added

**Review by calipee — APPROVED** [review]
Thanks for adding docs for VS Code. Good Work!

**Pull request merged**